modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/endofyear/EndOfYearManagerImpl.kt

```diff
@@ -59,48 +71,79 @@ class EndOfYearManagerImpl @Inject constructor(
         hasEpisodesPlayedUpto(YEAR, TimeUnit.MINUTES.toSeconds(EPISODE_MINIMUM_PLAYED_TIME_IN_MIN)) &&
             FeatureFlag.isEnabled(Feature.END_OF_YEAR_ENABLED)
 
-    /**
-     * Download the year's listening history.
-     */
+    private var synced: Double = 0.0
+    private var syncTotal: Double = 0.0
```

Comment:
Did you mean to put these above the `isEligibleForStories` function?

Reply:
Yes 🤦‍♀️. These are not needed in `EndOfYearManagerImpl` anymore as I moved them to `HistoryManager` in https://github.com/Automattic/pocket-casts-android/pull/1522/commits/e5d3b1e0b921b520556b300aabb2d06a3d518ebc.